### ISSUE TITLE
`eval $(podman-machine env box --varlink)` doesn't work

### ISSUE THREAD

**Issue by pixystone** [boot2podman/machine#13]
`$PODMAN_VARLINK_BRIDGE` is empty

```
[pixy@Pixys-MBP15.local /Users/pixy]
$eval $(podman-machine env box --varlink)

[pixy@Pixys-MBP15.local /Users/pixy]
$export | grep VARLINK
declare -x PODMAN_VARLINK_BRIDGE=""
declare -x VARLINK_BRIDGE="/usr/bin/ssh -F /dev/null -o ConnectionAttempts=3 -o ConnectTimeout=10 -o ControlMaster=no -o ControlPath=none -o LogLevel=quiet -o PasswordAuthentication=no -o ServerAliveInterval=60 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@127.0.0.1 -o IdentitiesOnly=yes -i /Users/pixy/.local/machine/machines/box/id_rsa -p 55180 varlink bridge"

[pixy@Pixys-MBP15.local /Users/pixy]
$podman ps
Error: error creating libpod runtime: dial unix /run/podman/io.podman: connect: no such file or directory
```

**Comment by pixystone** [comment]
I found this commit f8a0517, but not released yet.

**Comment by afbjorklund** [comment]
Right, the workaround is setting it to the other variable (after the env call)

**Comment by afbjorklund** [comment]
I think I will remove `$VARLINK_BRIDGE` completely, and make `--varlink` into a no-op and default.

It was never popular with upstream, they want `--bridge` to be explicit and seem to prefer bridge now.

**Comment by afbjorklund** [comment]
Fixed in v0.16 (16aed9e)
